ui/admin/tests/acceptance/host-catalogs/hosts/list-test.js

```diff
@@ -69,33 +66,28 @@ module('Acceptance | host-catalogs | hosts | list', function (hooks) {
     urls.newHost = `${urls.hosts}/new`;
     await authenticateSession({});
   });
+
   test('Users can navigate to hosts with proper authorization', async function (assert) {
     await visit(urls.hostCatalog);
+
     assert.ok(
       instances.hostCatalog.authorized_collection_actions.hosts.includes(
         'list',
       ),
     );
-    assert.dom(MANAGE_DROPDOWN_SELECTOR);
-    assert.ok(find(`[href="${urls.hosts}"]`));
+    assert.dom(selectors.MANAGE_DROPDOWN_HOST_CATALOG).exists();
+    assert.dom(commonSelectors.HREF(urls.hosts)).exists();
   });
 
   test('User cannot navigate to index without either list or create actions', async function (assert) {
     instances.hostCatalog.authorized_collection_actions.hosts = [];
     await visit(urls.hostCatalog);
+
     assert.notOk(
       instances.hostCatalog.authorized_collection_actions.hosts.includes(
         'list',
       ),
     );
-    assert.notOk(find(`[href="${urls.hosts}"]`));
-  });
-
-  test('User can navigate to index with only create action', async function (assert) {
```

Comment:
non-blocking: wondering why we removed this test?

Reply:
Follow [this thread](https://github.com/hashicorp/boundary-ui/pull/2629#discussion_r1884265981)for th full context.

Spoilert alert, I will add changes tomorrow, or when I have the time, to this PR.